libs/model/src/middleware/authorization.ts

```diff
@@ -194,39 +197,45 @@ async function isTopicMember(
 
   if (auth.topic.group_ids?.length === 0) return;
 
+  // check if user has permission to perform "action" in 'topic_id'
+  // the 'topic_id' can belong to any group where user has membership
+  // the group with 'topic_id' having higher permissions will take precedence
```

Comment:
if we are going with the higher permissions policy, the default group should be more restrictive

Reply:
I was thinking about the default group that will be created when we refactor the addresses table, but that's in the future